cves/CVE-2015-8218.yml

```diff
@@ -342,7 +365,15 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
-  answer:
+  answer: |
+    This vulnerability stemmed from, or at the very least was discovered as a
+    result of refactor. The vulnerable function did not exist at all prior to the
+    VCC, but the code did - it was just duplicated within another function. The
```

Comment:
I'm not sure I quite agree with your response here - it was pulled out of a different function, but from what I can see the code was basically copy/pasted into its own method. The vulnerability was there before then too, so what's the mistake that lead to that? Was this even a big mistake (in thinking of the timeline and severity of the vulnerability)?

Reply:
The original function had the protection. In the words of the commit message, the fix was to add a missing check

Reply:
Coming back to this after some additional investigation. The original, original commit of this file (and where I originally thought the code came from) did not have the vulnerability, but it also didn't have the functionality. I'm not sure if I misinterpreted the git blame or what exactly happened there, but that assumption derailed the rest of the investigation. I've updated the relevant places to better reflect what actually happened